tideways/vvv-header.php

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
this file should be named `tideways-header.php` or something more tideways specific

Reply:
It would be best to avoid the rename and have it keep the same filename

Reply:
renamed